Create readme

newellev opened the pull request:
Create readme and folder for Hardware Design files

newellev committed:
Create readme

Create readme and folder for Hardware Design files

newellev committed:
Create Readme

newellev committed:
Add files via upload

Kicad design files

newellev committed:
Create Readme

newellev committed:
Add files via upload

waxters-osu commented:
Run the design rules checker for your PCB. When I run it gives some errors for some capacitors (C2 through C5) overlapping solder masks between different nets. This is for the footprint, CAPC0603X33N. Itd be best to fix it; a short would suck.

newellev commented:
Common error for 0603 footprints in Kicad; Solder mask should not cause any short. This is based on research and not personal experience -- if you have information to support the opposite I will stand by.

waxters-osu commented:
> Common error for 0603 footprints in Kicad; Solder mask should not cause any short. This is based on research and not personal experience -- if you have information to support the opposite I will stand by.

Yeah see [this](https://forum.kicad.info/t/6-99-drc-fail-solder-mask-aperture-bridges-items-with-different-nets/35311/3) discussion. I'm good with ignoring it, but we should mark it as ignored (right click on it and "exclude this violation") and commit that.

newellev commented:
I did that on my version when I made it initially. So since it didn't come with what you downloaded I'm guessing it doesn't transfer.

waxters-osu commented:
> I did that on my version when I made it initially. So since it didn't come with what you downloaded I'm guessing it doesn't transfer.

I made the changes on my end. I think its in the `.kicad_prl` file which is local project settings from what I'm reading. I'll double check now.

newellev committed:
Adjusted Edge cuts

Adjusted Edge cuts and excluded DRC

waxters-osu committed:
fix: ignore DRC overlap soldermask 0603 footprint

waxters-osu committed:
chore(gitignore): add gitignore for kicad files in hw design

waxters-osu commented:
Alright I just made that change and added a gitignore for the the kicad junk files we don't want to commit. Should be good to merge now.

newellev committed:
Updated footprint library

Made footprint library the appropriate format such that pulling files does not require as much work for a clean DRC

newellev committed:
Returned PCB Files

Previous commit accidentally ommitted footprint files. This change rectifies this
